examples/github-repositories-custom-plugin/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "@algolia/github-repositories-custom-plugin",
+  "description": "Autocomplete example with custom GitHub Repositories plugin",
+  "version": "1.0.0-alpha.44",
+  "private": true,
+  "license": "MIT",
+  "main": "index.html",
+  "scripts": {
+    "build": "parcel build index.html",
+    "start": "parcel index.html"
+  },
+  "dependencies": {
+    "@algolia/autocomplete-js": "1.0.0-alpha.44",
+    "@algolia/autocomplete-theme-classic": "1.0.0-alpha.44",
+    "debounce-promise": "3.1.2",
+    "preact": "10.5.13",
+    "unfetch": "4.2.0"
+  },
+  "devDependencies": {
+    "@types/debounce-promise": "3.1.3",
```

We don't pin `@types` in our repos:

```suggestion
    "@types/debounce-promise": "^3.1.3",
```